src/tests/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 set(TESTS unit-tests.cpp)
+set(SOURCES ../app/doctor/doctor.cpp ../app/person/person.cpp ../app/visit/visit.cpp)
+set(CMAKE_OSX_SYSROOT "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk")
```

Comment:
Following my previous comment, if this is variable is it necessary? I do not see use of it.

Reply:
It seems not portable. I am even surprised it compiled in Docker container. 
If we cannot get rid of it by other means we should at least protect it if "if" statements for correct platform.